lib/court_data_adaptor/query/base.rb

```diff
@@ -17,6 +17,12 @@ def self.call(term, options = {})
       end
 
       def call
+        resource.with_headers("X-Request-ID": Current.request_id) do
+          make_request
+        end
+      end
+
+      def make_request
```

Comment:
this is confusing me, this should be in the other PR, maybe you forgot it? ... or is it just you need to rebase this? 🤔